### DISCUSSION TITLE
I need to compile server information to the client, but it doesn't compile correctly using GitHub

### DISCUSSION THREAD

**Discussion by ghost** [rustdesk/rustdesk#9891] [Q&A] [answered]
Since I don't have a Linux server, my compilation using GitHub's online compilation function always fails, and I need to compile my local server information into the client.

The remote control software in mainland China is too expensive for me, this project is good for me

**Comment by ghost** [comment]
I am from China, I don't know if we can discuss this in Chinese.

**Reply by rustdesk** [reply, marked as answer]
#763 Sorry, we do not waste time to help anyone for this.
